CRAYSAT-1417: SAT documentation RPM (#9)
Cray-HPE/docs-sat: rpm/README.md

> # System Admin Toolkit Offline Documentation

Author: @Benjamin-Rohlfing-HPE - thought you might want to review this file, it's documentation specific to the offline format of the docs. Any feedback you have would be welcome.

Reviewer: @ekamin-hpe thanks for the ping. Taking a look.

The SAT documentation can be found online at the following url:
https://cray-hpe.github.io/docs-sat

> Additionally, it is available offline in markdown format, which may be viewed
> with a markdown viewer or simply with a text editor. A table of contents can be
> found below.

Reviewer: ```suggestion
It is also available offline as markdown, which may be viewed with a markdown viewer or with a text editor. A table of contents can be
found below.
```

Reviewer: By "markdown viewer" we mean something that can render markdown, right?

Author: Correct. The suggestion looks good to me, fixed with: 22d69ae182d7bcc288d76a4bc0afb2848ee9ea29

The offline documentation is available in the `docs/` directory of the SAT
release distribution as well as in RPM package format. The RPM package is
installed as a part of the Ansible plays launched by the Configuration
Framework Service (CFS). Its files are installed to `/usr/share/doc/sat`.

## Check for Latest Documentation

Acquire the latest documentation RPM. This may include updates, corrections,
and enhancements that were not available until after the software release.

> 1. Check the version of the currently installed SAT documentation. This RPM

Reviewer: Nite: could you move the second sentences in each of these steps into the step body? Looks cleaner to me to keep the step titles to one sentence each.

Author: Fixed with 22d69ae182d7bcc288d76a4bc0afb2848ee9ea29

   will only be installed if the SAT CFS configuration layer has been applied.

   ```
   rpm -q docs-sat
   ```

2. Download and upgrade the latest documentation RPM. For example, to download
   and upgrade to the latest SAT 2.4 documentation RPM:

   ```
   rpm -Uvh https://artifactory.algol60.net/artifactory/sat-rpms/hpe/stable/sle-15sp3/docs-sat/2.4/noarch/docs-sat-latest.noarch.rpm
   ```

   If this machine does not have direct internet access, then this RPM will

>    need to be externally downloaded and copied to the system. This example
>    copies it to `ncn-m001`.

Reviewer: Nit

```suggestion
   need to be externally downloaded and copied to the system. The following example
   copies it to `ncn-m001`.
```

Author: Fixed with 22d69ae182d7bcc288d76a4bc0afb2848ee9ea29

   ```
   wget https://artifactory.algol60.net/artifactory/sat-rpms/hpe/stable/sle-15sp3/docs-sat/2.4/noarch/docs-sat-latest.noarch.rpm -O docs-sat-latest.noarch.rpm

>    scp -p docs-sat-*rpm ncn-m001:/root

Reviewer 2: Probably should drop the wildcard here and use the full `docs-sat-latest.noarch.rpm` just in case they've got other files matching the wildcard in their current directory.
